src/components/layer-controller/RasterLayerController.js

```diff
@@ -199,25 +150,20 @@ export default function RasterLayerController({
       update.domain = domains;
       update.slider = sliders;
     }
-    dispatch({
-      type: 'CHANGE_GLOBAL_CHANNELS_PROPERTIES',
-      layerId,
-      handleLayerChange,
-      payload: {
-        update,
-        publish: mouseUp,
-      },
-    });
+    const newChannels = channels.map(c => ({ ...c, ...update }));
+    if (mouseUp) {
+      setChannels(newChannels);
+    }
```

Comment:
I think we need something more advanced here (separate from the not-working aspect) - the controller doesn't work like it used to.  We can chat offline about this or I can take this on.  Basically, there needs to be some local state so the UI can update but then the actual layer only updates on `mouseUp`